front-end/e-learning/src/components/categories/List.js

```diff
@@ -132,6 +132,7 @@ const List = ({ listCategory, categories, token, user }) => {
             <TableBody>
               {categories
                 .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                .reverse()
```

Comment:
@rdelrosariojr We can also use `orderBy('desc')` in BE to fetch in latest first order. Just information, not a comment :)

Reply:
@AubreyV yes ma'am, I also mentioned it during our scrum meeting. I used .reverse() since I also use the API to display categories for users. But now that I modified Category display for users, I might try `orderBy('desc')`. Thank you for the info ma'am :)